Firestore/core/src/local/leveldb_key.cc

```diff
@@ -596,6 +612,14 @@ class Writer {
     WriteLabeledString(ComponentLabel::DocumentId, document_id);
   }
 
+  void WriteBundleId(absl::string_view bundle_id) {
```

Comment:
Preserve relative ordering and put this after `WriteSnapshotVersion`?

Reply:
Done.